packages/uikit/src/contexts/MatchBreakpoints/useMatchBreakpointsContext.ts

```diff
@@ -1,14 +1,13 @@
 import { useContext } from "react";
 import { MatchBreakpointsContext } from "./Provider";
+import { useMatchBreakpoints } from "../../hooks";
 
 const useMatchBreakpointsContext = () => {
   const matchBreakpointContext = useContext(MatchBreakpointsContext);
+  const contextAvailable = matchBreakpointContext !== undefined;
+  const hookState = useMatchBreakpoints(contextAvailable);
 
-  if (matchBreakpointContext === undefined) {
-    throw new Error("Match Breakpoint context is undefined");
-  }
-
-  return matchBreakpointContext;
+  return contextAvailable ? matchBreakpointContext : hookState;
```

Comment:
Could I ask: in which case, the component doesn't have context?

Reply:
Hmm. Basically, we add a production code for storybook. I think we could remove the hookState and add context in storybook. It would be more consistent.